src/test/e2e/09_component_compose_test.go

```diff
@@ -5,97 +5,116 @@
 package test
 
 import (
+	"context"
 	"fmt"
-	"os"
 	"path/filepath"
+	"runtime"
 	"testing"
 
+	goyaml "github.com/goccy/go-yaml"
 	"github.com/stretchr/testify/require"
-	"github.com/stretchr/testify/suite"
-)
-
-type CompositionSuite struct {
-	suite.Suite
-	*require.Assertions
-}
 
-var (
-	composeExample        = filepath.Join("examples", "composable-packages")
-	composeExamplePath    string
-	composeTest           = filepath.Join("src", "test", "packages", "09-composable-packages")
-	composeTestPath       string
-	composeTestBadLocalOS = filepath.Join("src", "test", "packages", "09-composable-packages", "bad-local-os")
+	layout2 "github.com/zarf-dev/zarf/src/internal/packager2/layout"
 )
 
-func (suite *CompositionSuite) SetupSuite() {
-	suite.Assertions = require.New(suite.T())
+func TestComposabilityExample(t *testing.T) {
+	t.Parallel()
 
-	// Setup the package paths after e2e has been initialized
-	composeExamplePath = filepath.Join("build", fmt.Sprintf("zarf-package-composable-packages-%s.tar.zst", e2e.Arch))
-	composeTestPath = filepath.Join("build", fmt.Sprintf("zarf-package-test-compose-package-%s.tar.zst", e2e.Arch))
-}
+	// Skip for Windows until path separators in packages are standardized.
```

Comment:
Is there a todo or an issue we can link in here

Reply:
Created issue #3275 for this.